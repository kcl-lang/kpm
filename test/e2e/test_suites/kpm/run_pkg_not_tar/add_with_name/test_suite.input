kpm add k8s