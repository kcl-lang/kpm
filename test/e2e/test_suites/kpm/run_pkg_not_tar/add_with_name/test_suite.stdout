kpm: adding dependency 'k8s'.
kpm: the lastest version '1.27' will be added.
kpm: downloading 'test/k8s:1.27' from 'localhost:5001/test/k8s:1.27'.
kpm: add dependency 'k8s' successfully.
